node/yarn-error.log

```diff
@@ -1,34 +1,34 @@
 Arguments: 
```

Comment:
can't we remove this file and add it to `.gitignore`?

Reply:
Sure !